doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -29,16 +30,18 @@ Ceph cluster enables the throttling of the operations(IOPS) belonging to
 different client classes (background recovery, scrub, snaptrim, client op,
 osd subop)”*.
 
-The purpose of the mclock profile is to help with the following,
+The purpose of the mclock profile is to help with the following:
 
-- Based on the capacity limits provided and the mclock profile selected,
-  determine the low level mclock resource control parameters,
-- Transparently apply the lower level mclock resource control parameters and
-  some Ceph configuration parameters depending on the profile.
+- the mclock profile uses the capacity limits provided within it and the mclock
+  profile that you select to determine the low-level mclock resource control
+  parameters
+- the mclock profile transparently appies the lower-level mclock resource
+  control parameters and, depending on the profile, some Ceph-configuration
+  parameters.
 
-The lower level mclock resource control parameters are the *reservation, weight
-and limit* that provide control of the resource shares as already described in
-the `OSD Config Reference`_ section.
+The low-level mclock resource control parameters are the *reservation*,
+*limitation*, and *weight* that provide control of the resource shares, as
```

Comment:
The code base uses "limit". Therefore, replace "limitation" with "limit".

Reply:
Done.